### ISSUE TITLE
Update runsc version used by gVisor addon 

### ISSUE THREAD

**Issue by ianlewis** [kubernetes/minikube#4482]
Now that minikube can use containerd 1.2+ we should update containerd to use shimv2 and update the runsc version used.

**Comment by ianlewis** [comment]
Currently blocked by #3512 since runsc uses pivot_root.

**Comment by medyagh** [comment]
Thanks for creating this issue, if you don't mind could you provide more context on the reason for this request? I would be happy to review any PR that would improve the gvsior addon

**Comment by ianlewis** [comment]
There isn't a specific reason besides that the version currently used is 6mo+ old and containerd isn't set up to use RuntimeClass and shim v2. It's been on my back burner for a while, just haven't gotten around to it. Added an issue to track. You can assign me me if inclined.

**Comment by afbjorklund** [comment]
Theoretically one could make that other program respect the legacy/vulnerable setting of `--no-pivot`.

**Comment by ianlewis** [comment]
@afbjorklund Yeah, Though since gVisor is a sandbox runtime I'd like to avoid adding adding less secure options if possible.

**Comment by tstromberg** [comment]
Merged in by #4494.
